core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/layout/dot-edit-layout/dot-edit-layout.component.html

```diff
@@ -3,6 +3,7 @@
     <dotcms-template-builder-lib
         [layout]="pageState.layout"
         [themeId]="pageState.template.theme"
+        [templateIdentifier]="templateIdentifier()"
```

Comment:
Why don't just pass as a prop this inside the layout? it will be simpler and more centralized.

Reply:
The templateIdentifier does not come within the layout
Comes inside the `template`
I could manually modify the layout and add the templateId there, but I didn't because I didn't want to modify the Layout interface
I found it easier to go directly to another prop

Should I still modify the layout?

Reply:
you can pass the whole template in that case and there you get the identifier and the themeID

Reply:
Yees, now i add both as template in prop
But that changes break the test and trigger selects in store
WIP to fix that
If there is other suggestion, please let me know

Reply:
Added!